content/style-guide/ui-components.md

```diff
@@ -0,0 +1,68 @@
+# UI components
+
+Our guidelines follow the [Google Developer's Style Guide on UI elements and interactions](https://developers.google.com/style/ui-elements).
+
+## When referring to UI elements, match capitalization, punctuation, and other formatting elements as the UI
```

Comment:
This heading sounds a little awkward to me, would "**to** the UI" or "**with** the UI" make more sense?